src/renderer/src/assets/main.css

```
@@ -2008,6 +2008,78 @@
+  order: 99;
+}
+
+/* Peek title filename/dirname open the shown file on click. */
+.monaco-editor .peekview-title .orca-peek-open-target {
+  cursor: pointer;
+}
+
+.monaco-editor .peekview-title .orca-peek-open-target:hover {
+  color: #58a6ff;
+  text-decoration: underline;
+  text-decoration-color: color-mix(in srgb, currentColor 45%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)